src/bench/bench_bitcoin.cpp

```diff
@@ -51,6 +51,13 @@ int main(int argc, char** argv)
     std::string scaling_str = gArgs.GetArg("-scaling", DEFAULT_BENCH_SCALING);
     bool is_list_only = gArgs.GetBoolArg("-list", false);
 
+    if (evaluations == 0) {
+        return EXIT_SUCCESS;
+    } else if (evaluations < 0) {
+        tfm::format(std::cerr, "Error parsing evaluations argument: %s\n", std::to_string(evaluations).c_str());
```

Comment:
the `.c_str() `should not be needed here

Reply:
also, casting evaluations from an integer to a string is wrong, it will do some pointer arithmethic, I think?

Why not
```c++
tfm::format(std::cerr, "Error parsing evaluations argument: %d\n", evaluations));
```

Reply:
Ah, ofc. Fixed :+1: